src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/SolutionGraph2SolutionMappings.java

```diff
@@ -0,0 +1,20 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper;
+
+import java.util.List;
+
+import org.apache.jena.rdf.model.Model;
+
+import se.liu.ida.hefquin.engine.data.SolutionMapping;
+import se.liu.ida.hefquin.engine.query.SPARQLQuery;
+
+public interface SolutionGraph2SolutionMappings {
```

Comment:
Also find a more suitable name here. See that the only method executes a SPARQL Query over a Model.

I'm also now curious if there is simply a Jena funcionality/utility that can be used instead of implementing this method.